akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -1550,14 +1550,50 @@ abstract class TimerGraphStageLogic(_shape: Shape) extends GraphStageLogic(_shap
    */
   final protected def schedulePeriodically(timerKey: Any, interval: FiniteDuration): Unit =
     schedulePeriodicallyWithInitialDelay(timerKey, interval, interval)
-
 }
 
+/** Java API: [[AbstractTimerGraphStageLogic]],alias with [[TimerGraphStageLogic]]*/
+abstract class AbstractTimerGraphStageLogic(_shape: Shape) extends TimerGraphStageLogic(_shape: Shape) {
```

Comment:
since the GraphStage api is not separated in javadsl and scaladsl I think I prefer to just add these as overloaded methods to `TimerGraphStageLogic`

Reply:
Ok~